assignments/this.js

```diff
@@ -1,26 +1,42 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window Binding is the default behavour where this references the window or global/console
+* 2. Implicit binding occors within the constructor function and where this references the proporties of the constructor function. 
```

Comment:
in terms of how it's being used in this assignment, yes, but remember it also works outside of constructor functions. specifically, anywhere there is a `this` pointer function preceded by a `.`, the `this` will reference whatever is left of the dot